### DISCUSSION TITLE
Why my Arctic Code Vault Contributor badget disappear

### DISCUSSION THREAD

**Discussion by VoiceBeer** [community/community#23869] [New to GitHub]
<p>I remember that I had that</p>
<blockquote>
<p>Arctic Code Vault Contributor</p>
</blockquote>
<p>name before, but after the achievement feature has come out, I found that I could not set the badge. :&lt;</p>
<p>Any help pls? Or I’m out? :&lt;</p>

**Comment by jfullstackdev** [comment]
<p>because it’s a badge already! and that is awesome,</p>
<p>check this out:</p>
<aside class="quote quote-modified" data-post="1" data-topic="175404">
  <div class="title">
    <div class="quote-controls"></div>
    
    <a href="https://github.community/t/new-awesome-badge-a-message-of-thanks/175404">New &amp; Awesome Badge, A Message Of Thanks!</a> <a class="badge-wrapper  bullet" href="/c/github-help/how-to-use-git-and-github/21"><span class="badge-category-parent-bg" style="background-color: #6a737d;"></span><span class="badge-category-bg" style="background-color: #ffd1ac;"></span><span style="" data-drop-close="true" class="badge-category clear-badge" title="Ask and answer questions regarding Git and GitHub.">How to use Git and GitHub</span></a>
  </div>
  <blockquote>
    <a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/f/8/f8993be5db3b4d8fa58b77bc84197a2c1f08792b.png" data-download-href="/uploads/short-url/ztcIkgQeDvFMuQKS9ZtoXUn10Fl.png?dl=1" title="image" rel="noopener nofollow ugc">[image]</a> 
the new Arctic Code Vault Contributor badge is really awesome, hoping for more badges that GitHub gives 
thanks a lot GitHub!
  </blockquote>
</aside>

**Comment by canuckjacq** [comment]
<p>:wave: Welcome! <a class="mention" href="/voicebeer">@VoiceBeer</a></p>
<p>I can see the badge on your profile - it just changed to a brighter image so you might not recognise it.</p>

**Comment by VoiceBeer** [comment, marked as answer]
<p>Hi thx and nice to mee ya :&gt;</p>
<p>I just re-added the email that is linked to the Arctic Code Vault Program, and the badge re-appears. :grin:</p>

**Reply by PixeliGer** [reply]
I have discovered that the badge also disappears from my profile, in the Email Settings part (found in the Access section) I can only see the option to add a new email or delete the existing one (but this warns that all association will be lost from my commits to that account)

**Reply by skdevelopers** [reply]
My badge is also disappeared where should i put my email ?

**Comment by mbiesiad** [comment]
<p>Hi, actually I have the same issue now :stuck_out_tongue: Where have you re-add the email?</p>
<p>Thanks in advance. Best wishes,</p>

**Comment by VoiceBeer** [comment]
<p>Hi <a class="mention" href="/mbiesiad">@mbiesiad</a> , just go to ur email settings (in the settings of your profile), and add the email account that you deleted before :&gt;, hope it works</p>

**Comment by faha1999** [comment]
<p>I had the <strong>Arctic Code Vault Contributor</strong> badge last 3 hours ago but now when I checked my profile I can’t see the Arctic Code Vault Contributor badge.</p>
<p>Already re-added the email my email address but till now the badge doesn’t re-appears.</p>
<p>What do I need to do?<br>
<img src="https://user-images.githubusercontent.com/54038576/181079780-a3b20734-792b-4429-8671-0fea6d55a3e0.jpeg" alt="Screenshot from 2021-06-25 07-55-46" data-base62-sha1="lX89fiKhpCNZCh8KmxjmGqYjMTc" width="327" height="411"></p>

**Comment by PeopleInside** [comment]
<p>I need help for the similar issue: my badge disappeared.<br>
I <a href="https://github.community/t/loosed-artic-contributor-badge/201060">reported here</a>.</p>
<p>I hope in some help with a solution from GitHub.<br>
Thanks.</p>

**Comment by bellodavid** [comment]
Add  the email that is linked to the Arctic Code Vault Program, and the badge re-appears.

**Comment by PixeliGer** [comment]
I also recently discovered that the badge disappeared from my profile but I have not been successful with the option to re-add my email

**Reply by PeopleInside** [reply]
You have to contact GitHub privately by opening a ticket.

**Comment by Fuatorium** [comment]
You can use these link: https://www.youtube.com/watch?v=oJWFfp3sGYA&t=348s&ab_channel=Thinkright20

**Comment by TheYali1** [comment]
d

**Comment by solartes** [comment]
Where is my lovely badge? I have added all the emails I have previously used.
